docs/DeveloperGuide.md

```diff
@@ -36,3 +233,32 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+## Command Summary
+
+| Action                                                | Format                                                                     |
+|-------------------------------------------------------|----------------------------------------------------------------------------|
+| Adding entries for meals                              | `log m/<MEAL_NAME> s/<SERVINGS>`                                           |
+| Adding entries for water                              | `log w/<AMOUNT_OF_WATER>`                                                  |
+| Adding entries for exercises                          | `log e/<EXERCISE_NAME> d/<DURATION>`                                       |
+| Showing daily summary of food, water intake and goals | `summary`                                                                  |
+| Setting daily calories goal                           | `set goal c/<CALORIE AMOUNT>`                                              |
+| Setting daily water goal                              | `set goal w/<WATER AMOUNT>`                                                |
+| View daily calorie intake                             | `show c`                                                                   |
+| View daily water intake                               | `show w`                                                                   |
+| View daily goals                                      | `show g`                                                                   |
+| View all entries logged                               | `list`                                                                     |
+| Getting help                                          | `help`                                                                     |
+| Find entries that contain the keyword                 | `find <KEYWORD>`                                                           |
+| Prints all food data                                  | `show food`                                                                |
+| Prints all exercise data                              | `show exercises`                                                           |
+| Deletes the water/meal entry                          | `delete <AMOUNT_OF_WATER>/<MEAL_NAME>`                                     |
+| Clears all entry                                      | `clear`                                                                    |
+| Adding new items to the food database and log meal    | `add m/<MEAL_NAME> c/<CALORIES_PER_SERVING(kCal)> s/<NUMBER_OF_SERVINGS>`  |
+| Adding new exercise to database and log it:           | `add e/<EXERCISE> c/<CALORIES_BURNT_PER_MIN(kCal)> d/<DURATION_PER_MIN>`   |
```

Comment:
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/111181615/480f6e7f-4989-454b-84f5-e73eb9b42c5c)

This portion is very well done and gives a good summary of the functionalities of the app